doc/source/contributing.rst

```diff
@@ -137,11 +137,57 @@ following the steps in the `Creating a New Pull Request
 section <#creating-a-new-pull-request>`__.
 
 
+Making Issues
+~~~~~~~~~~~~~
+Should you come across a bug in pyansys or otherwise encounter some
```

Comment:
```suggestion
Should you come across a bug in ``pymapdl`` or otherwise encounter some
```
pyansys is now the umbrella project including pyaedt, dpf, pymapdl, and pymapdl-reader